packages/uniforms-bridge-graphql/src/GraphQLBridge.ts

```diff
@@ -135,6 +136,13 @@ export default class GraphQLBridge extends Bridge {
       }
     }
 
+    if (isEnumType(fieldType)) {
+      const values = fieldType.getValues();
+      props.allowedValues = values.map(option => option.value);
+      props.transform = (value: unknown) =>
+        values.find(searchValue => searchValue.value === value)!.name;
+    }
```

Comment:
The code is good overall, but I'd rather see it as an alternative to the `options` declaration, i.e., add it as the last fallback. Otherwise, the schema will always override props.